tableauserverclient/models/flowrun_item.py

```diff
@@ -0,0 +1,106 @@
+import xml.etree.ElementTree as ET
+from ..datetime_helpers import parse_datetime
+import itertools
+
+
+class FlowRunItem(object):
+    def __init__(self) -> None:
+        self._id=None
+        self._flow_id=None
+        self._status=None
+        self._started_at=None
+        self._completed_at=None
+        self._progress=None
+        self._background_job_id=None
+
+    
+    @property
+    def id(self):
+        return self._id
+
+
+    @property
+    def flow_id(self):
+        return self._flow_id
+
+
+    @property
+    def status(self):
+        return self._status
+
+
+    @property
+    def started_at(self):
+        return self._started_at
+
+
+    @property
+    def completed_at(self):
+        return self._completed_at
+
+
+    @property
+    def progress(self):
+        return self._progress
+
+
+    @property
+    def background_job_id(self):
+        return self._background_job_id
+
+
+    def _set_values(
+        self,
+        id,
+        flow_id,
+        status,
+        started_at,
+        completed_at,
+        progress,
+        background_job_id,
+    ):
+        if id is not None:
+            self._id = id
+        if flow_id is not None:
+            self._flow_id = flow_id
+        if status is not None:
+            self._status = status
+        if started_at is not None:
+            self._started_at = started_at
+        if completed_at is not None:
+            self._completed_at = completed_at
+        if progress is not None:
+            self._progress = progress
+        if background_job_id is not None:
+            self._background_job_id = background_job_id
+
+
+    @classmethod
+    def from_response(cls, resp, ns):
+        all_flowrun_items = list()
+        parsed_response = ET.fromstring(resp)
+        all_flowrun_xml = itertools.chain(
```

Comment:
I'm not super familiar with flow runs -- is this because we have two versions of response or something?

Reply:
Yes, the `Get Flow Run` and `Get Flow Runs` endpoints seem to return different XML. (The documentation is also a little wrong about the XML document returned. It returns `flowId` and not `flow id`.

https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_flow.htm#get_flow_run

Reply:
@dzucker-tab FYI re
> The documentation is also a little wrong about the XML document returned. It returns flowId and not flow id.